aws-lambda-layer/Dockerfile

```diff
@@ -1,14 +1,19 @@
-FROM lambci/lambda:build-ruby2.5
+FROM amazonlinux:latest
```

Comment:
I think we probably want to use `amazonlinux:2` to avoid being surprised if/when AL3 comes out